var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+# 1) The CMake options exposed by `blaspp` allow for a value called `auto`. The
+#    value is not needed here as the choice of dependency in the spec
+#    determines the appropriate flags.
+#
+# 2) BLASFinder.cmake handles most options. For `auto`, it searches all blas
+#    libraries listed in `def_lib_list`.
+#
+# 3) ?? Custom blas library can be supplied via `BLAS_LIBRARIES`.
+#
+class Blaspp(CMakePackage):
+    """BLAS++: C++ API for the Basic Linear Algebra Subroutines (University of
+    Texas)."""
+
+    homepage = "https://bitbucket.org/icl/blaspp"
+    hg       = "https://bitbucket.org/icl/blaspp"
+    maintainers = ['teonnik', 'Sely85']
+
+    version('develop', hg=hg, revision="5191c9d")
+
+    variant('gfort',
```

Comment:
Are you aware if there are more potential interfaces, e.g. LLVM's upcoming one or other compilers?
We could make this a named variant then to support potential future options better than in a bool switch.

Reply:
I am not aware of any other interfaces at the moment.